lib/aws-eni/interface.rb

```diff
@@ -181,6 +181,7 @@ def info
               raise Errors::MetaBadResponse unless Meta.interface(hwaddr, '', not_found: nil)
               {
                 hwaddr:       hwaddr,
+                instance_id:  Meta.instance('instance-id'),
```

Comment:
It was easier to just add this to the info's metadata than trying to pull it from `lib/aws-eni.rb`.